core/src/main/scala/org/apache/spark/sql/crossdata/catalog/XDCatalog.scala

```diff
@@ -55,6 +55,11 @@ abstract class XDCatalog(val conf: CatalystConf = new SimpleCatalystConf(true),
     }
   }
 
+  def tableExistsInCatalog(tableIdentifier: Seq[String]): Boolean = {
+    val (table, database) = tableIdToTuple(tableIdentifier)
+    lookupTable(table, database).fold(false)(crossdataTable => true)
```

Comment:
Sadly, you can't do that because `lookupTable` is a method, not a function. You could make it a function with something like `lookupTable(_,_)` but the result is rather difficult to read :sob: